### ISSUE TITLE
Can we change our milestone naming convention?

### ISSUE THREAD

**Issue by uber5001** [Awk34/dotsynth#43]
With the names ".synth 0.1", ".synth 0.2", and ".synth 1.0", I don't really know where to put stuff. It's kinda like naming them "soon", "later", and "some day". Version names don't really have any meaning, other than "this version is new than that one".

Can we change our naming convention so that our milestone names match the set of features we want? I think it makes more sense to group the issues into similar groups by milestone, and then order them based on which is most important.

**Right now I don't want to worry about it**. I'd rather just keep coding. So don't go about changing all the names of the current milestones just because I complained. However, if a new major feature idea comes up, and we think we should work on it before moving too far ahead, can we make it a milestone?

One milestone I'm thinking of is "Sequencing".

**Comment by Awk34** [comment]
How about instead, change the description of the milestones to be how you would like the app to work at that point in time. When we get our first, usable app that performs like we would expect and would be happy to release it as '.synth 0.1', that milestone will be done. So 0.1 would be things like basic UI tweaks/enhancements, node-param connections, things that we would like to get done ASAP, etc.  Yes, it is kind of like "soon", "later", and "someday", but it gives us an Idea of how we would like the app to function at a specific state in the production timeline.

**Comment by uber5001** [comment]
Kk. I'm not too worried about it anyhoo.

**Comment by Awk34** [comment]
tl;dr get back to coding :smile:
